lib/table-storage/azure-table.mapper.ts

```diff
@@ -33,7 +33,7 @@ export class AzureEntityMapper {
 
     return result;
   }
-  static createEntity<D>(partialDto: Partial<AzureEntityMapper>, rowKeyValue = generateUuid()) {
+  static createEntity<D>(partialDto: Partial<AzureEntityMapper>, rowKeyValue: string) {
```

Comment:
I don't think we need to change this file. Actually `rowKeyValue = generateUuid()` means that if `rowKeyValue`is not provided, we will then call `generateUuid()` and assign it to `rowKeyValue`. This is exactly equivalent to the `if` statement added below 47. We can safely remove the changes from this file.

Reply:
excellent! changes removed.